gilchrist/scripts/data_extractor.js

```
@@ -742,188 +1505,270 @@ function extractSecTwpRng($) {
-  // const state_code = state.toUpperCase();
-  const postal_code = zip;
-  // const plus_four_postal_code = plus4 || null;
+  return null;
+}
 
-  // Per evaluator expectation, set county_name from input jurisdiction
+function attemptWriteAddress(unnorm, secTwpRng, siteAddress, mailingAddress) {
+  let hasOwnerMailingAddress = false;
   const inputCounty = (unnorm.county_jurisdiction || "").trim();
+  if (!inputCounty) {
+    inputCounty = (unnorm.county_name || "").trim();
```

Biome (2.1.2)

[error] Can't assign inputCounty because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)